packages/cashier/globals.d.ts

```diff
@@ -1,2 +1,3 @@
 declare module '@deriv/components';
 declare module '@deriv/shared';
+declare module '@deriv/p2p';
```

Comment:
@hamid-deriv We shouldn’t do this, instead should fix `package.json` in `p2p` package, Not sure why the main is set to `lib/index.js` :/

Reply:
@farzin-fs I think this needs to be fixed in the p2p package first and then remove from `global.d.ts` in the cashier package.

Reply:
@hamid-deriv Yeah you are right, We can move forward 🙇🏻